### PR TITLE
NIAD-769 Deploy gp2gp to ECS.

### PR DESCRIPTION
**Pull request by michalschott**

### CODE REVIEW

**Comment by LukaszKwasniewski** [comment]
Please add the cidr for gp2gp in the base component, the cidr are defined here: https://github.com/nhsconnect/integration-adaptors/blob/develop/terraform/aws/components/base/locals_cidrs.tf and an output for it here: https://github.com/nhsconnect/integration-adaptors/blob/develop/terraform/aws/components/base/outputs.tf SCR is not and will not be part of this code so you can simply rename SCR with gp2gp.

**Comment by LukaszKwasniewski** [comment]
Please also set defaults or values in etc/eu-west-2_build1.tfvars so we can run a successful plan: http://ec2-3-9-114-173.eu-west-2.compute.amazonaws.com/job/Terraform-test/282/console

**Comment by michalschott** [comment]
> Please add the cidr for gp2gp in the base component, the cidr are defined here: https://github.com/nhsconnect/integration-adaptors/blob/develop/terraform/aws/components/base/locals_cidrs.tf and an output for it here: https://github.com/nhsconnect/integration-adaptors/blob/develop/terraform/aws/components/base/outputs.tf SCR is not and will not be part of this code so you can simply rename SCR with gp2gp.

Done, cleanup / replace is out of scope.

> Please also set defaults or values in etc/eu-west-2_build1.tfvars so we can run a successful plan: http://ec2-3-9-114-173.eu-west-2.compute.amazonaws.com/job/Terraform-test/282/console

Done

File: terraform/Jenkinsfiles/TF_Scaffold.groovy [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
- 
+
 Map <String, Map<String, String>> componentImageBranch = [
-  OneOneOne: [ecrRepo: "111",   branch: "master"],
-  nhais:     [ecrRepo: "nhais", branch: "develop"],
-  nhais_responder: [ecrRepo: "nhais-fake-responder", branch: "origin-develop"]
+  OneOneOne:       [ecrRepo: "111",                  branch: "master"],
+  nhais:           [ecrRepo: "nhais",                branch: "develop"],
+  nhais_responder: [ecrRepo: "nhais-fake-responder", branch: "origin-develop"],
+  gp2gp:           [ecrRepo: "gp2gp",                branch: "develop"]
```
Comment (LukaszKwasniewski) [inline terraform/Jenkinsfiles/TF_Scaffold.groovy]: the table here is used to query ECR for latest image from 'master' branch. For gp2gp the 'master' branch is called 'main'. Without this terraform will try to set ` 067756640211.dkr.ecr.eu-west-2.amazonaws.com/gp2gp:null` as the image which will obviously fail

Comment (michalschott) [reply terraform/Jenkinsfiles/TF_Scaffold.groovy]: Sorted out.

**Review by LukaszKwasniewski — APPROVED** [review]

**Pull request merged**